x-pack/plugins/secops/server/graphql/kpi_network/schema.gql.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import gql from 'graphql-tag';
+
+export const kpiNetworkSchema = gql`
+  type kpiNetworkData {
```

Comment:
Nit: I believe GraphQL convention is to capitalize types -- so this would be `KpiNetworkData`.

Reply:
👍